libs/components/pages/src/lib/modules/needs-attention/needs-attention.component.ts

```diff
@@ -10,26 +25,64 @@ import { SkyActionHubNeedsAttention } from '../action-hub/types/action-hub-needs
   selector: 'sky-needs-attention',
   templateUrl: './needs-attention.component.html',
   styleUrls: ['./needs-attention.component.scss'],
-  standalone: false,
+  imports: [
+    SkyAppLinkModule,
+    SkyBoxModule,
+    SkyHrefModule,
+    SkyIconModule,
+    SkyPagesResourcesModule,
+    SkyThemeModule,
+    SkyTrimModule,
+    SkyWaitModule,
+    LinkAsModule,
+    NgClass,
+    AsyncPipe,
+  ],
 })
 export class SkyNeedsAttentionComponent {
-  @Input()
-  public set items(value: SkyActionHubNeedsAttention[] | undefined) {
-    this.#items = value;
-    if (value?.some((c: SkyActionHubNeedsAttention) => c.message)) {
-      this.#logService.deprecated(`SkyActionHubNeedsAttention.message`, {
-        deprecationMajorVersion: 7,
-        replacementRecommendation: 'Use `title` instead.',
-        moreInfoUrl:
-          'https://developer.blackbaud.com/skyux/components/action-hub?docs-active-tab=development#interface-skyactionhubneedsattention',
-      });
-    }
-  }
+  public readonly items = input<SkyActionHubNeedsAttention[]>([]);
 
-  public get items(): SkyActionHubNeedsAttention[] | undefined {
-    return this.#items;
-  }
+  protected readonly displayItems = toObservable<SkyActionHubNeedsAttention[]>(
+    this.items,
+  ).pipe(
+    switchMap((items) => {
+      if (!items?.length) {
+        return of([] as SkyActionHubNeedsAttention[]);
+      }
+      return forkJoin(
+        items.map((item) => {
+          if (item.permalink?.url?.includes('://')) {
+            if (!this.#resolver) {
+              this.#logService.error(
+                `SkyHrefResolverService is required but was not provided. Unable to resolve ${item.permalink.url}`,
+              );
+              return of(undefined);
+            }
+            return from(
+              this.#resolver.resolveHref({ url: item.permalink.url }),
+            ).pipe(map((result) => (result.userHasAccess ? item : undefined)));
+          }
+          return of(item);
+        }),
+      ).pipe(map((items) => items.filter(Boolean)));
+    }),
+    startWith([] as SkyActionHubNeedsAttention[]),
+  );
 
-  #items: SkyActionHubNeedsAttention[] | undefined;
-  #logService = inject(SkyLogService);
+  readonly #logService = inject(SkyLogService);
+  readonly #resolver = inject(SkyHrefResolverService, { optional: true });
```

Comment:
Making this optional so any tests do not start failing if they don't have `SkyHrefResolverService`. An error gets logged via `SkyLogService` if they did need resolver but didn't load it.